event_registration_multi_qty/models/event.py

```diff
@@ -0,0 +1,58 @@
+# -*- coding: utf-8 -*-
+# Copyright 2017 Sergio Teruel<sergio.teruel@tecnativa.com>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import api, fields, models
+
+
+class EventEvent(models.Model):
+    _inherit = 'event.event'
+
+    registration_multi_qty = fields.Boolean(
+        string='Registration Multi Qty',
+        help='Allow more than one registration'
+    )
+
+    @api.onchange('registration_multi_qty')
+    def _onchange_registration_multi_qty(self):
+        if not self.registration_multi_qty:
+            self.registration_ids.write({'qty': 1})
+
+    @api.multi
+    @api.depends('seats_max', 'registration_ids.state',
+                 'registration_ids.qty')
+    def _compute_seats(self):
+        for event in self:
+            if not event.registration_multi_qty:
+                return super(EventEvent, self)._compute_seats()
+            event.seats_unconfirmed = event.seats_reserved = 0
+            event.seats_used = event.seats_available = 0
```

Comment:
In these 2 lines you are actually performing 4 writes to DB. Later below, each `+=` call performs another write. How about working with simple variables that start with `0` and perform a single `.write({...})` at the end? 🤔